### PR TITLE
WID 725 app profile selector logic

### PR DESCRIPTION
**Pull request by andy-t-wang**
App selector and review status
Also add checks when we change the selector to verified to disable the input and show the verified data
Note the review status component is more specific than a general status bar but I figured that would be more practical 
Screenshot 2024-02-06 at 8 14 00 PM

Screenshot 2024-02-06 at 8 16 56 PM

### CODE REVIEW

File: web-app-router/next.config.js
Comment (andy-t-wang) [inline web-app-router/next.config.js:1]: Need to add to feature branch

File: web-app-router/components/ReviewStatus/index.tsx
Comment (andy-t-wang) [inline web-app-router/components/ReviewStatus/index.tsx:1]: This isn't really a general component specifically for the front page display and for the app profile so I added more specific logic to make it easier

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/Links/page/LinksForm/index.tsx [hunk 1]
```diff
@@ -81,6 +82,14 @@ export const LinksForm = (props: LinksFormProps) => {
     },
   });
 
+  useEffect(() => {
```
Comment (igorosip0v) [inline web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/Links/page/LinksForm/index.tsx:85]: @andy-t-wang What is this for? 
Let's left a comment here

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/StoreInfo/page/UpdateStoreInfoForm/index.tsx [hunk 2]
```diff
@@ -110,6 +115,14 @@ export const UpdateStoreInfoForm = (props: UpdateStoreInfoFormProps) => {
     },
   });
 
+  useEffect(() => {
```
Comment (igorosip0v) [inline web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/StoreInfo/page/UpdateStoreInfoForm/index.tsx:118]: Why do we need reset form on every dependencies change? 
Leave comment here too, please

**Review by igorosip0v — APPROVED** [review]

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/Links/page/LinksForm/index.tsx (hunk 1 above)
Comment (andy-t-wang) [reply web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/Links/page/LinksForm/index.tsx:85]: the form doesn't update with the new values when we change the selector

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/StoreInfo/page/UpdateStoreInfoForm/index.tsx (hunk 2 above)
Comment (andy-t-wang) [reply web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/StoreInfo/page/UpdateStoreInfoForm/index.tsx:118]: default values are not reactive so we use the hook in order to make it update when the view mode changes to verified

Comment (andy-t-wang) [reply web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/StoreInfo/page/UpdateStoreInfoForm/index.tsx:118]: But I updated to be specifically for those values

**Pull request merged**